### ISSUE TITLE
Build Next fails to do anything

### ISSUE THREAD

**Issue by lostfictions** [albertosantini/sublimetext-buildnext#7]
When hitting ctrl+b to build with Build Next on Windows, I get the following error:

```
Traceback (most recent call last):
  File "D:\Programs\Sublime Text 3\sublime_plugin.py", line 549, in run_
    return self.run(edit)
  File "BuildNext in C:\Users\s\AppData\Roaming\Sublime Text 3\Installed Packages\Build Next.sublime-package", line 268, in run
  File "BuildNext in C:\Users\s\AppData\Roaming\Sublime Text 3\Installed Packages\Build Next.sublime-package", line 183, in run
KeyError: 'C:\\Users\\s\\Haxe\\haxe\\lib\\luxe\\git\\tests\\features\\draw\\src\\Main.hx'
```

The build itself completes correctly, but the panel doesn't automatically close when there are no errors, nor is there any highlighting or gutter markers. The `goto_next_error` and `goto_prev_error` keybindings also do nothing.

My build file is pretty simple:

```json
{
	"shell_cmd": "haxelib run flow build web --project $project_path/project.flow",
	"working_dir": "${project_path:${folder}}",
	"file_regex": "^([^\\t\\n\\r]+.hx):(\\d+): characters (\\d+)-\\d+ : ([^\\t\\n\\r]+)",
	"env":
	{
		"ST_BUILD_ADJUST_COLUMNERROR": "1"
	}
}
```

**Comment by albertosantini** [comment]
Thanks for the feedback. A few notes for the troubleshooting.

Firstly I would run the build system *without* Build Next plugin.

If it works correctly, I would install Build Next plugin. Give a look at the README for a configuration  example. Notice `cmd` (vs. `cmd_shell`) and `"shell": true`.

However, I think, the build file is correct.

Then `file_regex` should parse the error line in `filename`, `line`, `column` and `error message`.

For instance,

```
{
    "shell_cmd": "C:\\My\\Programs\\haxe-3.1.3\\haxe -main $file_name --interp",
    "file_regex": "^([^\\t\\n\\r]+.hx):(\\d+): characters (\\d+)-\\d+ : ([^\\t\\n\\r]+)",
    "env":
    {
        "ST_BUILD_ADJUST_COLUMNERROR": "1"
    }
}
```
```
class HelloWorld {
  static public function main():Void {
    trace("Hello World")
  }
}
```
```
HelloWorld.hx:3: characters 23-24 : Missing ;
```

The issue is due to 
https://github.com/albertosantini/sublimetext-buildnext/blob/master/BuildNext.py#L68

Because the key of the error list is the fullpath of the file, while the `error_files` is only the filename.

I changed that line and it seems working correctly.
```
            indexes = [i for i, x in enumerate(error_files) if key.find(x) >= 0]
```

Shall you try that fix before committing a new release, please?

**Comment by albertosantini** [comment]
Released 1.3.1 version with the wrokaround.
If it doesn't fix the issue, please provide also an example of build output.
